src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Offender.kt

```diff
@@ -107,13 +107,14 @@ class OffenderBuilder(
     youthAdultCode: String,
     agencyLocationId: String,
     livingUnitId: Long,
+    bookingSequence: Int?,
     dsl: BookingDsl.() -> Unit,
   ) = bookingBuilderFactory.builder()
     .let { builder ->
       builder.build(
         offender = offender,
         // if you want multiple bookings then create them with latest booking first so it gets seq 1 like in Nomis
-        bookingSequence = offender.bookings.size + 1,
+        bookingSequence = bookingSequence ?: (offender.bookings.size + 1),
```

Comment:
A quick hack to allow us to specify the booking sequence - works around an issue with the DSL where the booking sequence on both the main booking and an alias booking are 1. This should go away when a proper fix is done (currently WIP).